straxen/test_utils.py

```diff
@@ -107,14 +107,31 @@ def _is_on_pytest():
     return 'PYTEST_CURRENT_TEST' in os_environ
 
 
+def _get_fake_daq_reader():
+    class DAQReader(straxen.DAQReader):
+        """
+        Dummy version of the DAQ reader to make sure that all the testing
+        data produced here will have a different lineage
+        """
+        __version__ = "MOCKTESTDATA"
+    return DAQReader
+
+
 def nt_test_context(target_context='xenonnt_online',
                     deregister=('peak_veto_tags', 'events_tagged'),
                     **kwargs):
+    if not straxen.utilix_is_configured(warning_message=False):
+        kwargs.setdefault('_database_init', False)
+
     st = getattr(straxen.contexts, target_context)(**kwargs)
     st.set_config({'diagnose_sorting': True})
-    st._plugin_class_registry['raw_records'].__version__ = "MOCKTESTDATA"  # noqa
+    st.register(_get_fake_daq_reader())
```

Comment:
I noticed that `st._plugin_class_registry['raw_records'].__version__ = "MOCKTESTDATA"  # noqa` caused lineage changes in some later tests too. This way, we don't change the lineage of the real DAQReader